### ISSUE TITLE
قطع شدن مکرر

### ISSUE THREAD

**Issue by sahmadiut** [radkesvat/RTCF#67]
روی نسخه ۱.۶ تست کردم با یکی دو تا یوزر خیلی خوب بود هم پینگش هم سرعتش ولی به محض اینکه چیزی میزنم برای دانلود یا هر دو دستگاه شروع به استفاده زیاد کردم قطع شد سریع. این کا رو دو سه بار انجام دادم هر با همینطوری شد.

![image](https://github.com/radkesvat/RTCF/assets/74642981/7e09c96e-549f-43c9-be89-a78e8a196460)

**Comment by mohamad15516** [comment]
برای منم این شکلی هست و متصل نمیشه اگر کمکی میکنه برای رفع ارور ها و باگ ها 
![Screenshot_20240115_060022_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/25d6f9e7-6d63-410b-a4c7-2e213f768e21)

![Screenshot_20240115_060053_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/1094db23-3a44-4a08-9cbd-3498f6a71c49)

**Comment by mohamad15516** [comment]
> برای منم این شکلی هست و متصل نمیشه اگر کمکی میکنه برای رفع ارور ها و باگ ها 
> خارج :
> ![Screenshot_20240115_060022_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/25d6f9e7-6d63-410b-a4c7-2e213f768e21)
> 
> ایران:
> 
> ![Screenshot_20240115_060053_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/1094db23-3a44-4a08-9cbd-3498f6a71c49)
> 
> 

 با تغییر ممابع سرور ایران به رم:۳ و cpu:3 core و ورژن ۱.۶ سرویس متصل شد با سرعت عالی حالا باید چک کنم ببینم قطعی و وصلی داره یا خیر اما یه چیزی که دیدم مشکل بالارفتن رم سرور خارج هست

**Comment by sahmadiut** [comment]
> روی نسخه ۱.۶ تست کردم با یکی دو تا یوزر خیلی خوب بود هم پینگش هم سرعتش ولی به محض اینکه چیزی میزنم برای دانلود یا هر دو دستگاه شروع به استفاده زیاد کردم قطع شد سریع. این کا رو دو سه بار انجام دادم هر با همینطوری شد.
> 
> ![image](https://private-user-images.githubusercontent.com/74642981/296634608-7e09c96e-549f-43c9-be89-a78e8a196460.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUyODkxMTEsIm5iZiI6MTcwNTI4ODgxMSwicGF0aCI6Ii83NDY0Mjk4MS8yOTY2MzQ2MDgtN2UwOWM5NmUtNTQ5Zi00M2M5LWJlODktYTc4ZThhMTk2NDYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE1VDAzMjAxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgyMzc1NGFiZDViNmI3MDJiMDIyMzEwNWQxYzNhMzA5ZmY5MDNmNTA0ZDJhNTA0NGEwZjc4N2ZiZDAyNjllNzUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sVj1CMNxH5myluXbyE0Gvg5W5KjiOXyhbmiXlYb4Lb4)

روی نسخه ۱.۷ هم همین الان تست کردم مشکل رفع نشد متاسفانه

**Comment by mohamad15516** [comment]
برای منم این شکلی هست و متصل نمیشه اگر کمکی میکنه برای رفع ارور ها و باگ ها 
![Screenshot_20240115_060022_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/25d6f9e7-6d63-410b-a4c7-2e213f768e21)

![Screenshot_20240115_060053_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/1094db23-3a44-4a08-9cbd-3498f6a71c49)

 توی نسخه ۱.۷ به راحتی وصل شد حتی با منابع کم دارم تست میکنم

**Comment by mohamad15516** [comment]
> برای منم این شکلی هست و متصل نمیشه اگر کمکی میکنه برای رفع ارور ها و باگ ها 
> ![Screenshot_20240115_060022_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/25d6f9e7-6d63-410b-a4c7-2e213f768e21)
> 
> ![Screenshot_20240115_060053_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/1094db23-3a44-4a08-9cbd-3498f6a71c49)
> 
>  توی نسخه ۱.۷ به راحتی وصل شد حتی با منابع کم دارم تست میکنم 

کماکان معلوم نیس مشکل از کجاست اما مدام قطع و وصل میشه ورژن ۱.۷ 
سرور ایران:



سرور خارج :
![Screenshot_20240115_071614_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/d24ab792-cc80-4a95-a2dc-57237c018f16)

**Comment by ali90500** [comment]
ارور نسخه 1.7 

fatal.nim(53)            sysFatal
port.nim(68)             read
port.nim(68)             read
Error: unhandled exception: port.nim(64, 9) `self.readPort == self.writePort`  [AssertionDefect]

**Comment by ali90500** [comment]
روی نسخه 1.7 وصل شدنش اوکیه، سرعت و اینا هم اوکیه ولی یه دفعه کرش میکنه خصوصا با دو سرویس

**Comment by Eslender73** [comment]
> روی نسخه 1.7 وصل شدنش اوکیه، سرعت و اینا هم اوکیه ولی یه دفعه کرش میکنه خصوصا با دو سرویس

میشه ۱.۸ رو تست کنید مارو در جریان بزارید

**Comment by mohamad15516** [comment]
هنوز مشکل مثل این که برقرار هست 
سرور ایران :
![Screenshot_20240115_222924_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/97896274-896c-42e7-b3c8-a6e8e85d8f4e)


سرور خارج :
![Screenshot_20240115_222843_Termius.jpg](https://github.com/radkesvat/RTCF/assets/113285931/bd7dfbff-b744-4225-a03c-f59501212aad)

**Comment by Obeid1375** [comment]
قطع و وصلیش اوکی شد توی نسخه 1.8 ؟

**Comment by sahmadiut** [comment]
> قطع و وصلیش اوکی شد توی نسخه 1.8 ؟

نه. بهتر شد ولی همچنان قطع میشه. سرعتش و پینگش عالی و آیپی کثیف رو هم راحت رد میکنه. فقط امیدوارم این مشکل قطع شدنش درست بشه دیگه عالی میشه

**Comment by Eslender73** [comment]
> > قطع و وصلیش اوکی شد توی نسخه 1.8 ؟
> 
> نه. بهتر شد ولی همچنان قطع میشه. سرعتش و پینگش عالی و آیپی کثیف رو هم راحت رد میکنه. فقط امیدوارم این مشکل قطع شدنش درست بشه دیگه عالی میشه

۱.۹ چطور؟ من سرورم تداخل داره نمیتونم تست کنم ببخشید میپرسم

**Comment by mohamad15516** [comment]
حل شده مشکلاتش دیگه مثل این که قطعی نداره
